vimode/README

```diff
@@ -413,6 +413,15 @@ a new command, please do not forget to update the table below.::
                                        cursor [into register x]
     y             ["x]y{motion}      yank Nmove text [into register x]
     yy            ["x]yy             yank N lines [into register x]
+    zA            zA                 open a closed fold or close an open fold
+                                       recursively
+    zC            zC                 close folds recursively
+    zM            zM                 set 'foldlevel' to zero
+    zO            zO                 open folds recursively
+    zR            zR                 set 'foldlevel' to the deepest fold
+    za            za                 open a closed fold, close an open fold
+    zc            zc                 close a fold
+    zo            zo                 open fold
```

Comment:
Could you move this block to the same place as in index.txt, i.e. under section "2.5 commands starting with z"? And also within this section to the same place as in index.txt? (of course taking into account that some commands are missing)